rules/math_big_rat.go

```diff
@@ -21,13 +21,12 @@ func (r *usingOldMathBig) Match(node ast.Node, ctx *gosec.Context) (gi *issue.Is
 		return nil, nil
 	}
 
-	confidence := issue.Low
 	major, minor, build := gosec.GoVersion()
 	if major == 1 && (minor == 16 && build < 14 || minor == 17 && build < 7) {
```

Comment:
Shouldn't be this? 🤔

```suggestion
	if major == 1 && (minor < 16 || minor == 16 && build < 14 || minor == 17 && build < 7) {
```

Reply:
Good catch! It should have been that way from the beginning, I assume.